### PR TITLE
[MOS-219] Expand "Setup IDE" to include QtCreator

### PR DESCRIPTION
**Pull request by GravisZro**
These are initial instructions on how to configure the QtCreator IDE.

### CODE REVIEW

**Comment by SP2FET** [comment]
Here's your ticket: MOS-219

File: doc/setup_ide.md [hunk 1]
```diff
@@ -133,6 +134,34 @@ After the above steps:
 - Change run/debug `Working directory` for Linux configuration. (`Edit configurations…` -> `PurePhone`)
 - Change `Working directory` from `…/PurePhone/build` to `…PurePhone/build-linux`
 
+### Setting up in QtCreator
+
+- Install `QtCreator` version `3.5` or later
+- Install `CMake` version `3.19.5` or later
+- It is highly recommened that to install `Ninja` as it will decrease rebuild times.
```
Comment (mjan-dev) [inline doc/setup_ide.md:141]:
```suggestion
- It is highly recommened to install `Ninja` as it will decrease rebuild times.
```

Comment (SP2FET) [reply doc/setup_ide.md:141]: reccomended* :D

Comment (GravisZro) [reply doc/setup_ide.md]: Thank you for the corrections, I have incorporated them.  I'm just glad we no longer live in the age of typewriters.

**Review by mjan-dev — APPROVED** [review]

**Review by pholat — APPROVED** [review]
thanks!

**Pull request merged**